README.md

```diff
@@ -99,6 +99,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - [Health Checks](doc/health-check.md): Enable health check on the upstream node, and will automatically filter unhealthy nodes during load balancing to ensure system stability.
   - Circuit-Breaker: Intelligent tracking of unhealthy upstream services.
   - [Proxy Mirror](doc/plugins/proxy-mirror.md): Provides the ability to mirror client requests.
+  - [Traffic Split](doc/plugins/traffic-split.md): Support the functions of gray release, blue-green release and custom release.
```

Comment:
Not so intuitive.

Use: allows users to incrementally direct percentages of traffic between various upstreams.

Reply:
updated.